src/test/cucumber/mnemonics.feature

```diff
@@ -40,26 +44,30 @@ Feature: Mnemonics
 		When I enter "sdk ug grails" and answer "n"
 		Then I see "Upgrade:"
 		And I see "grails (1.3.9 < 2.4.4)"
+		And the exit code is 0
 
 	Scenario: Shortcut for installing a Candidate Version
 		Given the candidate "grails" version "2.1.0" is not installed
 		And the candidate "grails" version "2.1.0" is available for download
 		And the system is bootstrapped
-		When I enter "sdk i grails 2.1.0" and answer "Y"
```

Comment:
I believe that in this situation, the candidate was not previously installed, so do not have to answer.